quicklogic/qlf_k4n8/devices/umc22/qlf_k4n8-qlf_k4n8_umc22_24x24.csv

```diff
@@ -0,0 +1,960 @@
+orientation,row,col,pin_num_in_cell,port_name,mapped_pin,GPIO_type

```

Comment:
Can this be generated and/or imported from the relevant part database?  I don't believe this kind of data really belongs in arch-defs?

Reply:
I am adding this file just for running test cases so that I can refer to the mapped pin in pcf file present in the testcases. In customer scenario, they are going to specify this file as an input to symbiflow with their pin-mapping to the device interface ports.